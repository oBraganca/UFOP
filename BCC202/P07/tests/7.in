116 18 1 108 50 90 58 65 106 78 23 81 49 107 103 91 105 12 101 31 33 48 94 110 79 9 112 15 22 89 21 61 60 45 66 70 16 40 54 82 115 92 10 4 114 26 28 93 88 30 38 46 51 63 20 71 43 35 27 42 47 24 113 99 95 11 69 39 72 29 73 34 100 80 53 2 96 19 84 25 5 41 86 111 55 97 52 3 83 68 13 37 62 75 67 59 116 57 76 64 8 6 44 14 56 74 85 98 32 109 7 104 77 102 87 17 36
154 133 21 124 45 58 70 17 106 141 64 97 68 26 89 52 59 48 128 91 16 54 65 15 74 149 115 100 76 140 57 28 113 8 104 72 41 67 154 122 131 9 35 105 86 120 142 51 5 46 112 43 94 63 145 138 23 22 109 125 20 135 102 87 13 62 11 121 85 7 123 44 71 19 92 82 73 37 78 139 53 25 143 95 153 27 1 101 118 49 90 151 50 77 116 129 137 24 111 84 134 80 107 4 114 152 56 66 130 93 88 136 55 12 69 79 38 6 96 98 144 14 119 132 3 126 117 40 108 99 10 61 31 146 39 47 30 75 83 34 18 60 103 42 110 29 150 147 127 2 148 81 33 36 32
129 86 34 19 5 1 70 23 66 129 40 54 68 115 25 4 120 65 69 109 59 104 47 37 29 113 97 83 42 46 85 102 114 80 126 22 79 92 96 36 39 94 87 6 58 117 74 98 20 95 101 76 73 24 89 52 32 12 84 17 57 41 9 110 50 125 124 100 53 2 63 99 45 88 7 27 48 3 30 75 67 91 56 38 107 14 93 122 61 31 16 112 108 72 128 10 78 13 81 55 15 119 127 123 33 28 82 43 51 121 21 118 49 18 62 77 71 105 116 8 60 44 11 111 103 90 64 26 106 35
141 110 87 62 105 102 21 122 26 89 132 111 120 7 140 82 67 25 134 101 14 114 10 119 121 42 81 55 93 124 49 8 58 41 106 73 80 24 22 78 139 50 23 37 84 18 39 98 88 12 71 11 83 9 129 94 4 96 31 72 46 85 32 77 115 104 107 52 108 40 100 13 126 112 43 29 64 5 60 136 48 91 35 118 131 137 127 30 53 141 15 63 125 76 34 86 59 57 3 33 123 99 74 103 1 17 36 97 79 113 117 54 69 61 128 133 66 51 44 20 135 95 2 138 90 6 109 116 75 38 56 47 27 28 16 70 68 65 19 130 45 92
97 19 15 45 18 90 93 38 37 42 60 22 89 6 71 72 62 4 61 44 67 21 76 75 53 56 10 79 49 7 69 55 66 59 63 5 30 20 74 14 83 70 35 64 29 24 65 8 94 96 80 47 50 11 58 43 9 97 36 52 91 88 27 12 31 54 17 2 68 41 84 57 95 46 26 32 28 48 16 92 39 34 3 86 82 77 13 51 78 87 40 73 81 25 85 23 1 33
192 153 75 189 25 179 61 191 22 49 18 70 176 68 114 145 21 168 39 6 142 33 27 118 159 17 170 150 119 20 41 55 102 76 190 149 171 167 105 53 83 8 141 28 3 48 19 92 79 87 47 88 181 91 131 147 121 38 4 110 116 54 9 172 124 98 1 143 24 115 163 104 26 161 108 103 77 180 32 106 14 132 128 109 31 129 72 29 97 81 42 157 94 46 139 80 84 82 69 74 50 57 188 23 34 5 7 138 169 140 64 152 56 184 36 156 35 112 66 186 144 113 96 16 40 89 178 111 151 11 162 12 60 78 58 127 126 155 123 71 185 148 154 158 173 101 160 175 135 15 44 174 166 10 62 117 136 137 95 52 133 51 130 63 13 90 120 165 30 73 45 187 100 192 177 85 183 2 125 164 37 134 43 65 99 93 182 59 146 86 122 107 67
109 13 10 104 88 21 92 81 99 72 47 27 95 48 71 34 69 28 45 41 7 9 24 5 16 59 14 101 55 61 29 3 44 80 20 49 43 63 79 106 31 56 36 90 96 17 97 50 91 23 108 77 85 64 30 32 89 40 25 94 26 105 15 35 70 39 75 11 100 102 33 76 98 74 8 93 1 51 57 84 42 62 67 60 66 82 19 22 12 53 107 4 6 58 73 18 65 78 37 86 54 87 103 83 109 46 52 68 2 38
81 5 62 17 1 11 29 6 65 36 80 31 39 21 78 10 13 54 9 4 28 43 32 81 69 22 12 47 37 53 74 7 59 41 30 44 16 24 73 63 67 79 75 68 76 34 55 71 42 15 60 26 19 8 45 51 72 27 49 40 50 48 70 33 61 58 18 3 23 20 57 14 25 64 38 77 52 35 46 66 56 2
111 9 83 1 2 58 50 80 36 71 33 38 85 3 56 79 55 57 91 31 74 93 19 84 73 102 39 53 42 46 98 52 7 25 54 86 5 107 32 4 15 99 47 62 16 27 24 72 17 67 78 104 43 100 60 69 94 110 18 20 61 70 105 101 76 22 41 49 6 59 109 92 12 37 40 65 8 96 23 63 89 81 97 77 13 95 64 11 45 75 21 34 30 106 68 108 35 44 28 29 10 48 51 90 82 14 26 111 66 103 88 87
159 16 91 148 118 155 98 7 22 130 75 158 51 27 143 144 45 83 26 132 119 133 55 36 13 38 149 108 14 105 78 54 69 86 18 9 125 151 146 66 46 85 70 47 114 157 30 76 49 53 112 48 150 117 25 1 24 59 68 31 107 5 37 73 84 23 110 4 145 100 82 136 99 147 41 42 77 15 88 43 109 115 153 28 17 32 81 122 97 64 123 52 56 35 34 141 113 140 50 120 159 12 39 111 2 142 137 131 72 67 102 33 3 126 154 57 129 6 62 92 40 104 156 90 127 134 135 95 20 101 93 96 11 89 139 74 87 10 21 103 128 79 152 60 121 71 29 65 116 94 106 138 63 8 124 61 44 58 80 19
55 4 30 54 1 25 32 15 14 49 11 52 51 12 41 38 28 42 20 46 5 22 40 37 36 29 39 3 47 55 48 18 6 53 19 34 23 33 50 27 26 44 21 35 8 43 10 45 2 17 13 16 9 31 24 7
155 147 133 40 110 103 48 23 15 30 6 89 84 31 121 63 108 55 83 116 47 115 73 87 94 81 12 154 148 45 129 86 29 35 22 19 146 61 143 42 24 80 60 78 44 69 17 3 21 65 106 124 59 58 153 136 111 64 92 51 127 99 8 71 66 62 134 96 57 74 79 97 135 131 43 54 67 10 85 95 5 93 126 130 20 123 128 155 46 122 27 77 100 98 4 13 105 2 53 118 139 137 49 50 120 91 32 76 38 149 88 104 70 1 39 119 151 145 37 112 132 9 68 11 36 101 75 140 28 142 107 14 102 152 34 109 144 82 33 138 7 56 72 113 16 52 26 141 117 90 114 18 41 125 150 25
14 6 10 9 12 2 14 13 7 5 8 4 1 11 3
187 44 67 89 64 47 82 108 38 58 8 53 164 25 144 71 128 112 179 9 69 21 4 87 10 143 57 56 22 146 11 168 118 52 43 90 7 37 142 153 94 172 105 127 122 66 70 166 95 15 86 68 73 100 149 46 28 187 19 129 61 16 150 123 141 158 34 148 55 98 145 171 131 59 99 12 157 49 121 65 134 177 173 124 30 182 91 151 63 51 97 54 106 101 178 84 80 116 36 109 185 26 126 62 154 147 140 83 42 41 115 135 163 76 33 136 119 110 132 1 6 176 79 107 165 152 117 20 137 24 96 72 31 23 5 50 180 114 39 181 175 93 85 125 74 18 130 29 138 14 40 77 92 133 169 48 88 174 111 3 2 35 161 32 27 104 167 156 45 81 155 184 120 102 75 60 162 183 159 113 170 17 186 139 160 103 13 78
166 59 28 95 45 4 144 159 126 156 58 29 44 14 43 117 63 112 103 99 10 108 146 148 105 19 18 38 163 57 15 9 27 79 141 55 94 17 83 24 128 54 81 98 49 143 118 127 129 138 34 75 136 157 150 161 106 65 16 135 12 53 47 151 56 80 152 122 154 25 119 36 130 64 155 142 60 69 115 84 26 61 153 7 101 20 72 3 107 165 52 46 77 33 6 116 30 139 114 35 5 40 145 73 11 48 123 23 8 91 71 39 22 88 100 62 134 78 97 132 120 133 74 124 113 87 160 13 164 96 51 42 90 37 102 137 111 76 70 104 140 66 110 2 158 21 50 1 32 92 162 68 86 67 121 93 166 85 31 131 109 149 125 82 147 89 41
162 162 34 7 49 153 45 39 101 156 43 36 139 106 140 61 1 14 147 126 95 8 40 66 111 70 121 120 67 62 3 122 149 154 58 82 141 5 44 42 107 89 91 4 135 23 55 87 110 137 93 17 47 26 133 119 132 73 96 148 86 112 13 155 6 21 63 29 65 83 31 145 81 104 11 152 48 138 41 144 50 27 151 118 128 38 54 115 78 52 79 85 46 37 146 94 143 130 9 25 12 77 68 30 90 99 18 2 159 102 108 64 16 158 113 19 136 127 10 24 97 105 98 74 35 20 150 72 75 134 117 123 59 84 114 53 131 129 60 80 56 124 109 100 57 32 103 116 28 22 157 51 92 88 161 33 69 160 142 15 125 71 76
113 42 73 34 90 111 46 43 24 78 80 44 16 33 95 113 102 112 67 1 84 71 56 19 4 39 17 98 29 7 2 9 60 6 93 105 104 68 103 94 55 52 109 47 79 64 99 11 35 27 87 85 76 81 65 92 75 5 58 62 22 51 100 59 82 21 12 83 48 108 28 50 106 14 25 8 97 10 36 63 37 77 38 70 101 96 53 91 13 86 3 15 41 31 54 23 30 32 40 66 61 49 20 26 72 110 107 89 69 74 45 57 88 18
196 40 19 15 152 163 95 36 83 106 145 100 157 150 23 81 91 86 29 180 136 50 88 111 125 93 33 172 2 154 139 103 141 71 8 147 158 7 68 60 90 69 102 123 13 82 67 11 52 170 112 63 176 55 70 65 169 6 149 75 105 131 87 186 20 165 59 37 54 24 117 38 126 156 3 189 42 196 116 174 72 184 193 45 49 190 47 51 146 167 194 58 43 46 14 161 128 57 133 28 137 84 109 132 110 164 135 127 9 30 104 21 148 16 4 187 120 77 144 27 121 85 44 22 160 142 78 113 96 129 41 188 138 107 175 181 35 92 159 179 191 34 173 17 10 162 62 18 64 124 171 143 155 32 12 98 99 140 114 151 177 79 153 89 1 166 74 48 94 80 66 118 39 192 182 53 101 119 25 56 185 134 183 61 115 130 31 5 178 122 168 108 76 97 195 73 26
46 25 28 27 20 1 42 38 15 6 43 16 30 22 31 46 45 34 8 9 37 19 33 4 41 13 2 18 29 44 21 10 14 23 26 39 17 11 3 5 40 32 35 7 12 36 24
18 13 10 17 9 12 8 16 7 4 11 14 1 15 6 3 18 5 2
197 72 120 29 169 134 165 167 33 160 102 108 151 54 127 55 51 27 133 15 104 82 174 137 162 168 65 166 157 17 116 98 32 110 12 13 131 6 89 73 2 26 126 3 105 145 147 163 196 87 64 100 113 164 46 92 69 7 53 185 71 140 189 31 195 181 11 146 158 138 170 35 149 179 187 28 91 36 66 114 5 59 141 61 109 142 111 112 176 155 48 47 62 123 171 44 1 156 124 4 9 183 96 132 144 20 194 129 34 83 78 177 77 148 118 94 21 95 41 136 153 70 39 68 119 159 24 182 150 40 154 106 193 19 178 22 81 37 23 184 93 90 45 192 79 191 103 63 101 117 58 18 43 190 197 86 99 10 76 16 80 172 49 115 139 161 84 88 25 8 125 186 85 188 74 42 180 57 175 152 75 97 67 122 52 14 135 173 56 128 121 130 38 107 143 60 30 50
175 56 17 117 113 101 108 154 29 37 57 127 103 26 2 12 134 159 68 166 69 24 147 172 109 65 21 94 167 150 84 140 149 49 59 105 155 173 39 70 106 121 32 13 152 27 73 88 138 145 151 83 40 126 48 153 128 123 120 60 30 97 47 160 38 66 55 86 89 36 76 78 80 99 129 164 5 175 91 161 14 136 23 141 62 100 87 31 156 41 85 51 42 125 170 139 52 67 116 75 111 7 53 28 144 34 61 148 90 15 165 104 82 25 118 122 54 157 71 110 50 43 92 162 142 107 9 20 8 146 16 18 63 133 64 58 171 74 112 33 77 19 81 163 131 124 168 3 95 169 115 79 174 6 35 158 114 135 1 130 10 119 137 22 132 4 45 96 143 46 102 72 98 44 93 11
99 92 67 91 66 6 8 70 56 1 60 69 34 76 3 22 98 26 81 44 86 9 39 52 74 33 35 80 97 48 61 49 93 38 54 94 11 40 88 75 30 55 64 36 29 32 13 83 18 47 27 71 24 79 77 85 73 96 51 89 45 90 20 14 17 65 87 46 50 2 43 12 16 25 42 58 68 28 41 82 5 10 57 95 99 78 84 4 19 62 7 15 72 53 31 23 21 59 37 63
130 80 82 77 15 59 16 90 56 91 39 10 109 92 9 99 53 49 73 36 34 68 65 86 12 106 89 100 111 104 119 61 125 78 31 124 121 72 130 123 126 19 48 74 42 1 30 4 47 79 55 25 37 41 35 44 116 60 50 93 75 26 3 71 118 17 28 67 24 29 117 18 101 103 21 69 110 58 46 122 2 113 5 66 114 27 64 22 81 83 57 108 115 33 95 87 6 23 20 54 63 13 7 129 62 70 107 120 14 128 43 76 112 52 88 94 98 84 45 51 102 38 8 11 32 40 127 105 85 96 97
67 14 57 40 27 67 47 53 4 28 41 24 5 63 56 17 54 58 25 9 66 18 29 1 16 23 42 45 51 50 11 2 60 44 15 13 38 31 55 21 52 61 20 36 59 33 46 19 62 12 30 7 35 10 26 8 65 34 39 3 43 6 49 37 32 22 64 48
37 23 29 20 33 11 8 6 24 18 14 2 34 30 31 15 19 22 13 5 4 27 10 28 7 16 12 17 25 1 3 9 37 35 21 26 36 32
61 24 14 1 9 34 19 43 51 26 36 32 59 30 5 20 18 60 45 39 46 35 11 61 13 47 23 50 27 58 29 38 42 48 53 2 15 55 31 44 41 52 7 25 21 8 4 12 54 40 3 10 57 49 33 37 16 22 28 56 6 17
48 39 22 32 9 11 15 16 1 37 18 28 17 47 2 34 31 6 44 40 30 23 27 10 48 41 46 38 19 45 26 42 35 13 25 3 36 12 24 8 20 21 29 14 33 5 43 4 7
30 8 4 30 29 5 16 22 3 6 24 2 15 12 28 11 14 13 27 1 17 26 23 18 10 7 9 25 21 19 20
117 34 51 66 115 91 50 101 78 40 31 6 4 94 33 71 93 95 15 1 11 23 37 113 62 88 69 63 96 102 90 30 85 97 16 72 99 112 116 9 70 86 65 56 36 105 14 89 20 21 103 27 80 12 111 117 19 82 57 84 25 60 44 74 76 87 5 64 77 32 26 52 92 43 109 35 47 110 75 68 39 41 108 107 17 53 8 3 45 98 104 114 46 28 13 73 100 54 10 42 22 81 61 48 83 55 18 7 106 29 38 67 59 58 24 79 49 2
0
