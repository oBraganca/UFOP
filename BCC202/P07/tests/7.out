Carlos
Carlos
Carlos
Marcelo
Marcelo
Marcelo
Carlos
Marcelo
Carlos
Carlos
Carlos
Marcelo
Marcelo
Marcelo
Carlos
Marcelo
Carlos
Carlos
Marcelo
Marcelo
Marcelo
Marcelo
Marcelo
Marcelo
Marcelo
Marcelo
Carlos
Marcelo
Marcelo
Carlos
